g2core/board/ArduinoDue/gShield-pinout.h

```diff
@@ -258,12 +258,12 @@ pin_number kADC14_PinNumber = -1;  // Not physially pinned out
 
 
 // GRBL / gShield compatibility pins -- Due board ONLY
-
+// NOTE: These dont appear to work
```

Comment:
Oh sorry I made a mistake its `INPUT_ACTION_HALT` not `INPUT_ACTION_HOLD`. I just tested and `INPUT_ACTION_STOP` appears to work properly and can be cleared with a `~`. `INPUT_ACTION_CYCLE_START` also works properly.